src/main/java/by/antonsh/lesson9/Ex28/ObjectEx.java

```diff
@@ -0,0 +1,55 @@
+package by.antonsh.lesson9.Ex28;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+/*
+Написать класс, который умеет хранить в себе массив любых типов данных.
+Реализовать метод get(int index), который возвращает любой элемент массива по индексу.
+ */
+
+/**
+ * This class create Object array and fill it.
+ */
+public class ObjectEx {
```

Comment:
we don't need this class, need only Generic

Reply:
Ok, i delete